src/custom-element.js

```diff
@@ -444,9 +444,6 @@ function createBaseCustomElementClass(win) {
       this.layout_ = Layout.NODISPLAY;
 
       /** @private {number} */
-      this.layoutWidth_ = -1;
```

Comment:
No, I think I'm looking at the right one. This appears to be `a9808200539a76eb965b6c6f7d0aba79cce8a4e1` commit...